### PR TITLE
[Outlook] (tutorial) Fix ModuleNotFoundError

### LINKED ISSUE #3242: ModuleNotFoundError after updating webpack config file
Prompted with ModuleNotFoundError after adding the following to the webpack.config.js file and running `npm run build` (see [error.txt](https://github.com/OfficeDev/office-js-docs-pr/files/8052936/error.txt) for full error log):
```
new HtmlWebpackPlugin({
  filename: "dialog.html",
  template: "./src/settings/dialog.html",
  chunks: ["polyfill", "dialog"]
})
```
### Specifications
**Document section**
'Update webpack config settings', steps 3-5.

**Platform**
Windows 11 Version 21H2 (OS Build 22000.493)

**Visual Studio Code** 
Version: 1.64.0 (user setup)
Commit: 5554b12acf27056905806867f251c859323ff7e9
Date: 2022-02-03T04:22:20.678Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.22000

**npm**
```
{
  'office-addin-taskpane-js': '0.0.1',
  npm: '8.4.0',
  node: '16.13.2',
  v8: '9.4.146.24-node.14',
  uv: '1.42.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '93',
  nghttp2: '1.45.1',
  napi: '8',
  llhttp: '6.0.4',
  openssl: '1.1.1l+quic',
  cldr: '39.0',
  icu: '69.1',
  tz: '2021a',
  unicode: '13.0',
  ngtcp2: '0.1.0-DEV',
  nghttp3: '0.1.0-DEV'
}
```

Was able to repro the error on a VM with the same specs.

### Other notes

The webpack config file generated through yo office doesn't match the code snippets provided in the 'Update webpack config settings' section.

- Step 1
  - Generated file doesn't contain `commands: "./src/commands/commands.js`.
  - Generated file's `polyfill` value is `"@babel/polyfill`.
- Step 2
  - Generated file's `plugins` array doesn't match the tutorial's code snippet:
    
image

---

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a59958d2-f428-9ca2-3440-a5af662386c0
* Version Independent ID: dcaf0b5b-08ea-c437-130b-342a1be62a16
* Content: [Tutorial: Build a message compose Outlook add-in - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/tutorials/outlook-tutorial)
* Content Source: [docs/tutorials/outlook-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/tutorials/outlook-tutorial.md)
* Product: **outlook**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

### PR DESCRIPTION
**Pull request by samantharamon**
Fixes #3242

### CODE REVIEW

File: docs/tutorials/outlook-tutorial.md [hunk 1]
```diff
@@ -875,7 +882,7 @@ function setConfig(config, callback) {
 
 ### Create new functions to process gists
 
-Next, open the **./src/helpers/gist-api.js** file and add the following functions. Note the following:
+Next, open the *./src/helpers/gist-api.js* file and add the following functions. Note the following:
 
 - If the gist contains HTML, the add-in will insert the HTML as-is into the body of the message.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:887]: Should be [as is](https://ahdictionary.com/word/search.html?q=as+is) - here and other instances
```suggestion
- If the gist contains HTML, the add-in will insert the HTML as is into the body of the message.
```

File: docs/tutorials/outlook-tutorial.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Tutorial: Build a message compose Outlook add-in'
 description: 'In this tutorial, you will build an Outlook add-in that inserts GitHub gists into the body of a new message.'
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:3]:
```suggestion
title: Tutorial: Build a message compose Outlook add-in
description: In this tutorial, you will build an Outlook add-in that inserts GitHub gists into the body of a new message.
```

File: docs/develop/develop-add-ins-vscode.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Develop Office Add-ins with Visual Studio Code
 description: 'How to develop Office Add-ins with Visual Studio Code.'
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/develop-add-ins-vscode.md:3]:
```suggestion
title: Develop Office Add-ins with Visual Studio Code
description: How to develop Office Add-ins with Visual Studio Code.
```

File: docs/testing/debug-desktop-using-edge-chromium.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Debug add-ins on Windows using Visual Studio Code and Microsoft Edge WebView2 (Chromium-based)
 description: 'Learn how to debug Office Add-ins that use Microsoft Edge WebView2 (Chromium-based) in VS Code.'
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-desktop-using-edge-chromium.md:3]:
```suggestion
title: Debug add-ins on Windows using Visual Studio Code and Microsoft Edge WebView2 (Chromium-based)
description: Learn how to debug Office Add-ins that use Microsoft Edge WebView2 (Chromium-based) in VS Code.
```

**Review by ElizabethSamuel-MSFT — CHANGES_REQUESTED** [review]
Great job investigating and fixing up the tutorial!

File: docs/tutorials/outlook-tutorial.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
-title: 'Tutorial: Build a message compose Outlook add-in'
-description: 'In this tutorial, you will build an Outlook add-in that inserts GitHub gists into the body of a new message.'
-ms.date: 01/06/2022
+title: Tutorial: Build a message compose Outlook add-in
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:2]: Sorry, missed the colon here so you need the quotes
```suggestion
title: 'Tutorial: Build a message compose Outlook add-in'
```

**Comment by samantharamon** [comment]
Thanks for making corrections and pointing me in the right direction, @ElizabethSamuel-MSFT. Additional formatting changes have been made for your review.

File: docs/tutorials/outlook-tutorial.md [hunk 6]
```diff
@@ -176,21 +179,21 @@ Now that you've verified the base add-in works, you can customize it to add more
 
 ### Remove the MessageReadCommandSurface extension point
 
-Open the **manifest.xml** file and locate the `ExtensionPoint` element with type `MessageReadCommandSurface`. Delete this `ExtensionPoint` element (including its closing tag) to remove the buttons from the read message window.
+Open the **manifest.xml** file and locate the **ExtensionPoint** element with type **MessageReadCommandSurface**. Delete this **ExtensionPoint** element (including its closing tag) to remove the buttons from the read message window.
 
 ### Add the MessageComposeCommandSurface extension point
 
-Locate the line in the manifest that reads `</DesktopFormFactor>`. Immediately before this line, insert the following XML markup. Note the following about this markup.
+Locate the line in the manifest that reads **</DesktopFormFactor>**. Immediately before this line, insert the following XML markup. Note the following about this markup.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:186]:
```suggestion
Locate the line in the manifest that reads `</DesktopFormFactor>`. Immediately before this line, insert the following XML markup. Note the following about this markup.
```

File: docs/tutorials/outlook-tutorial.md [hunk 7]
```diff
@@ -176,21 +179,21 @@ Now that you've verified the base add-in works, you can customize it to add more
...
 
-- The `ExtensionPoint` with `xsi:type="MessageComposeCommandSurface"` indicates that you're defining buttons to add to the compose message window.
+- The **ExtensionPoint** with **xsi:type="MessageComposeCommandSurface"** indicates that you're defining buttons to add to the compose message window.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:188]:
```suggestion
- The **ExtensionPoint** with `xsi:type="MessageComposeCommandSurface"` indicates that you're defining buttons to add to the compose message window.
```

File: docs/tutorials/outlook-tutorial.md [hunk 8]
```diff
@@ -176,21 +179,21 @@ Now that you've verified the base add-in works, you can customize it to add more
...
 
-- By using an `OfficeTab` element with `id="TabDefault"`, you're indicating you want to add the buttons to the default tab on the ribbon.
+- By using an **OfficeTab** element with **id="TabDefault"**, you're indicating you want to add the buttons to the default tab on the ribbon.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:190]:
```suggestion
- By using an **OfficeTab** element with `id="TabDefault"`, you're indicating you want to add the buttons to the default tab on the ribbon.
```

File: docs/tutorials/outlook-tutorial.md [hunk 9]
```diff
@@ -176,21 +179,21 @@ Now that you've verified the base add-in works, you can customize it to add more
...
 
-- The `Group` element defines the grouping for the new buttons, with a label set by the `groupLabel` resource.
+- The **Group** element defines the grouping for the new buttons, with a label set by the **groupLabel** resource.
 
-- The first `Control` element contains an `Action` element with `xsi:type="ShowTaskPane"`, so this button opens a task pane.
+- The first **Control** element contains an **Action** element with **xsi:type="ShowTaskPane"**, so this button opens a task pane.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:194]:
```suggestion
- The first **Control** element contains an **Action** element with `xsi:type="ShowTaskPane"`, so this button opens a task pane.
```

File: docs/tutorials/outlook-tutorial.md [hunk 10]
```diff
@@ -176,21 +179,21 @@ Now that you've verified the base add-in works, you can customize it to add more
...
 
-- The second `Control` element contains an `Action` element with `xsi:type="ExecuteFunction"`, so this button invokes a JavaScript function contained in the function file.
+- The second **Control** element contains an **Action** element with **xsi:type="ExecuteFunction"**, so this button invokes a JavaScript function contained in the function file.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:196]:
```suggestion
- The second **Control** element contains an **Action** element with `xsi:type="ExecuteFunction"`, so this button invokes a JavaScript function contained in the function file.
```

File: docs/tutorials/outlook-tutorial.md [hunk 11]
```diff
@@ -877,7 +884,7 @@ function setConfig(config, callback) {
 
 Next, open the **./src/helpers/gist-api.js** file and add the following functions. Note the following:
 
-- If the gist contains HTML, the add-in will insert the HTML as-is into the body of the message.
+- If the gist contains HTML, the add-in will insert the HTML as is into the body of the message.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/outlook-tutorial.md:887]: I think there's "as-is" somewhere else in this file - can you check?

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**